web/build/resources/main/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url = jdbc:mysql://localhost:3306/buy_certificate_rest_dev?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC
+spring.datasource.username = root
```

Comment:
datasource, jpa properties should be in corresponding module "repository"

Reply:
made : moved to  "repository"